Azure.OpenAI/Client/Program.cs

```diff
@@ -14,5 +16,10 @@
 builder.Services.AddSpeechSynthesisServices();
 builder.Services.AddSpeechRecognitionServices();
 builder.Services.AddMudServices();
+builder.Services.AddLocalization();
+
+var host = builder.Build();
+
+await Localization.ConfigCulture(host);
```

Comment:
```suggestion
await host.ConfigCulture();
```

Use it as an extension method.